recipes/core/tests/test_models.py

```diff
@@ -0,0 +1,37 @@
+# from unittest.mock import patch
+
+from django.test import TestCase
+
+from core.models import Recipe, Ingredient
+
+
+class ModelTests(TestCase):
```

Comment:
IMO no need to add tests for discrete things Django does. Tests are good if there is logic in the model, or you want to test some sort of dynamic behavior between models.

Reply:
Understood!